src/helpers/common.ts

```diff
@@ -1,9 +1,23 @@
 import { JRPCResponse } from "@toruslabs/constants";
 import { Ecies } from "@toruslabs/eccrypto";
+import { BN } from "bn.js";
+import { ec as EC } from "elliptic";
 import JsonStringify from "json-stable-stringify";
 
-import { CommitmentRequestResult, EciesHex, VerifierLookupResponse } from "../interfaces";
+import { CommitmentRequestResult, EciesHex, KeyType, VerifierLookupResponse } from "../interfaces";
+import { keccak256 } from "./keyUtils";
 
+export const ed25519Curve = new EC("ed25519");
```

Comment:
define this inside the function for better tree shaking

Reply:
done